website/docs/d/outputs.html.markdown

```diff
@@ -9,10 +9,8 @@ description: |-
 This data source is used to retrieve the state outputs for a given workspace.
 It enables output values in one Terraform configuration to be used in another.
 
-The `values` attribute has been defined as sensitive because the workspace
-outputs may contain sensitive values and the provider protocol does not support
-marking values as [sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) after this information is known. You can access
-the subset of non-sensitive outputs using the `nonsensitive_values` attribute.
+~> **NOTE:** The `values` attribute is preemptively marked sensitive and is only populated after a run completes on the associated workspace. To access non-sensitive values, use the `nonsensitive_values` attribute, which is a subset of outputs
```

Comment:
I think it might be helpful to retain the link to the sensitive documentation:

```suggestion
~> **NOTE:** The `values` attribute is preemptively marked [sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) and is only populated _after_ a run completes on the associated workspace. To access non-sensitive values, use the `nonsensitive_values` attribute, which is a subset of outputs.
```

With regards to @Uk1288 's comment, I think it's clearer to have the second sentence as is, starting with `To access non-sensitive values...` because it answers the question that appeared in my mind when reading the doc, how do I access non-sensitive values?

Reply:
I'm going to take @Uk1288's suggestion because it also corrects from passive voice to active voice